package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -606,15 +608,32 @@ def _read_next_timestep(self):
 
         coord_cols = convention_to_col_ix[self.lammps_coordinate_convention]
 
+        # pass possible additional fields from LAMMPS dump
+        # pass velocities
+        velocity_col_names = ["vx", "vy", "vz"]
```

Comment:
I would make this a class attribute, much like `self._conventions` is, as the data is not mutable or instance dependent. :)